Meadowlark-js/backends/meadowlark-aws-dynamodb-opensearch/src/BaseDynamoRepository.ts

```diff
@@ -4,21 +4,22 @@
 // See the LICENSE and NOTICES files in the project root for more information.
 
 import R from 'ramda';
-import invariant from 'invariant';
+import { invariant } from 'ts-invariant';
```

Comment:
ts-invariant lets type-narrowing work with invariants e.g. asserting not-null removes null from type possibilities afterwards